src/screens/OtherProfile/index.js

```diff
@@ -652,14 +660,14 @@ const styles = StyleSheet.create({
   wrapImageProfile: {
     marginTop: 24,
     flexDirection: 'column',
-    backgroundColor: 'red'
+    backgroundColor: COLORS.redalert
   },
   nameProfile: {
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 17,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color and text color properties have been updated to use the new `COLORS` object from the `theme` module. This is a good practice for maintaining consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->